packages/node_modules/nav-frontend-toggle/src/toggle-gruppe.tsx

```diff
@@ -0,0 +1,157 @@
+import * as React from 'react';
+import * as PT from 'prop-types';
+import * as cn from 'classnames';
+import { omit, guid } from 'nav-frontend-js-utils';
+
+import ToggleKnappPure, { ToggleKnappPureProps, ToggleKnappPurePropsShape } from './toggle-knapp-pure';
+
+import 'nav-frontend-toggle-style';
+
+export interface ToggleGruppeProps {
+    /**
+     * Array av toggle knapper, se `toggle-knapp-pure.tsx`
+     */
+    toggles?: ToggleKnappPureProps[];
+    /**
+     * Egendefinert klassenavn.
+     */
+    className?: string;
+    /**
+     * Hvis `true` vil det være mulig å velge flere knapper om gangen.
+     */
+    multi?: boolean;
+    /**
+     * Angir hvilke knapper som skal være trykket inn som default.
+     */
+    defaultPressed?: number | number[];
```

Comment:
Kan ikke dette styres gjennom `pressed`-propen på `ToggleGruppeKnappPure` alene? Skummelt å styre dette med tallverdier i en liste som gis gjennom props. Hvis vi har to måter å gjøre det på, vil det dessuten sannsynligvis ende opp med at noen setter `defaultPressed` på `ToggleGruppe` og en `pressed`-prop på `ToggleKnappPure`, og ikke skjønner hvorfor det ikke fungerer som forventet. Keep it simple 😄